### PR TITLE
Change parameters to explicitly pass in `genesis_validators_root` and `context`

### PR DESCRIPTION
**Pull request by EchoAlice**

### CODE REVIEW

File: mev-relay-rs/src/relay.rs [hunk 1]
```diff
@@ -616,17 +624,14 @@ impl BlindedBlockRelayer for Relay {
 fn verify_blinded_block_signature(
     auction_request: &AuctionRequest,
     signed_block: &mut SignedBlindedBeaconBlock,
-    relay: &Relay,
+    genesis_validators_root: &Node,
```
Comment (ralexstokes) [inline mev-relay-rs/src/relay.rs:627]:
```suggestion
    genesis_validators_root: &Root,
```

File: mev-relay-rs/src/relay.rs [hunk 2]
```diff
@@ -9,7 +9,7 @@ use ethereum_consensus::{
     crypto::SecretKey,
     deneb::mainnet as deneb,
     primitives::{BlsPublicKey, Epoch, Root, Slot, U256},
-    ssz::prelude::Merkleized,
+    ssz::prelude::{Merkleized, Node},
```
Comment (ralexstokes) [inline mev-relay-rs/src/relay.rs:12]: let's just use `Root` below

**Review by ralexstokes — CHANGES_REQUESTED** [review]
let's use the `Root` type instead of `Node` and also you'll need to rebase from `main`

File: mev-relay-rs/src/relay.rs (hunk 1 above)
Comment (EchoAlice) [reply mev-relay-rs/src/relay.rs:627]: Soooo picky... 👀 😴

**Review by ralexstokes — APPROVED** [review]
looks good!

**Pull request merged**